client/state/selectors/get-comments-page.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+
+/**
+ * Returns a list of comment IDs for the requested page.
+ *
+ * @param {Object} state Redux state.
+ * @param {Number} siteId Site identifier.
+ * @param {Number} page Requested page.
+ * @param {String} status Comments status filter.
+ * @param {Number} [postId] Post identifier.
+ * @param {String} [search] Search query.
+ * @returns {Array} List of comment IDs for the requested page.
+ */
+export const getCommentsPage = createSelector(
```

Comment:
I think we can avoid using memo here. We're not doing anything expensive here are we?

eg it's mostly 
```
get( state, [ 'ui', 'comments', 'queries', siteId, parent, filter, page ], [] )
```